### DISCUSSION TITLE
Question About Browsing the Database

### DISCUSSION THREAD

**Discussion by GeorgeHJ** [jcorporation/myMPD#41] [Q&A] [answered]
Hello,

I am having trouble using the Browse Database feature.

As you can see it isn't separating out artists.

![screenshot_2018-08-30_19-00-27](https://user-images.githubusercontent.com/30212436/44870062-ffae2c80-ac86-11e8-9e3f-8ac798b3def2.png)

Browsing the filesystem works as expected (similar to ympd).

Sorry for not using the template but I get the feeling this isn't a bug but rather something I'm doing wrong.

How should I go about trouble shooting?

Thanks.

ps. if you think this merits a bug report I will make one

**Comment by jcorporation** [comment]
Does ``mpc list Albumartist`` works in your environment?

See you any error in the Firefox Web Console?

**Comment by Emilot** [comment]
You may have an ad blocker in Firefox. Disable it and try again :)

**Comment by GeorgeHJ** [comment]
Thanks for being so helpful both.
Unfortunately disabling my adblocker doesn't change matters.
However I can't list the AlbumArtist tag with MPC. Is this a compile time option I need to find and change? I'm using an AUR package on Arch Linux Arm.

**Comment by jcorporation** [comment, marked as answer]
You must enable AlbumArtist in your mpd.conf.

``metadata_to_use "albumartist,artist,album,title,track,name,genre,date"``

And then rebuild your mpd database - stop mpd, delete it, start mpd and run ``mpc update``
Afterwards listing AlbumArtists should work.

**Comment by GeorgeHJ** [comment]
It works!

Huzzah!

I did just as you said. Restarting MPD with systemctl did the trick. The database was checked and deleted by MPD on its startup.

Thanks! Sorry to bother you, I did have an exhaustive look at mpd documentation but it can be difficult to use sometimes.
